common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -531,13 +531,13 @@ def test_download_transcript_button_works_correctly(self):
         self.assertTrue(self.video.downloaded_transcript_contains_text('txt', 'Welcome to edX.'))
 
         # open video "B"
```

Comment:
The video names are accurate, they just don't correspond to the way we are navigating the page. Will update to clarify.